### DISCUSSION TITLE
前世，今生，青梅，朱马

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#3010] [其他]
![mmexport1724038057524](https://github.com/user-attachments/assets/c1cde1b1-0517-401b-b17b-5781a26d919a)
午安，新艾利都，昨晚做了个奇怪的梦，梦里的前辈身着古装，腰间别着长剑，穿梭在竹林中，而我的身体只是本能的跟着她…跟着她…，就像平时执行任务一样形影不离。直到她唤出我的名字，我才突然惊醒，意识到这只是一个梦。

我把这事和前辈说了，她却笑我想象力丰富…说的也是，真是一个奇怪的梦啊…

不过，或许在前世的时候，我和前辈就已经是互相依赖的搭档了呢…

> 图源：pixiv 末那一时 （太喜欢这个作品了，已经换成电脑壁纸了）（朱鸢和前辈的情谊赛高！）

**Comment by RaphaelHider** [comment]
我在RandomPlay看过一部录像带，讲的是世界之外有很多相似的其他世界，各个世界有一些相似的人，他们性格、样貌甚至名字都一样，却走着不一样的人生轨迹，有着截然不同却同样精彩的故事。也许你梦到的就是另一个世界的你和前辈呢~

**Reply by ZhuYuan-148** [reply]
今天值完勤就去录像店借一盘看看！

**Comment by EDEN54418** [comment]
很有反差感的组合，直如朱丝绳，清如玉壶冰，来形容非常贴切呢。

**Reply by ZhuYuan-148** [reply]
虽然前辈在治安局有些松弛，但毕竟是老前辈。在执勤方面也是很有正义感的。

**Comment by Abandon12393** [comment]
话说我前天在工地上高塔上看见青衣治安官了，只有她一个人进了空洞，也没见她带着邦布，还是说青衣治安官已经把萝卜内置在系统里了吗，希望能平安回来吧。
